### DISCUSSION TITLE
lobe-chat和其他chatgpt的回答对比

### DISCUSSION THREAD

**Discussion by czxin788** [lobehub/lobehub#1206] [General]
下面是lobe-chat的回答：
![image](https://github.com/lobehub/lobe-chat/assets/13861904/9fcba1ee-a437-4f1b-9962-7d9ffb5cf227)

我用了另外 一款开源gpt，我觉得这个解释更加详细
![image](https://github.com/lobehub/lobe-chat/assets/13861904/b862e1b0-44cf-496c-9e84-b83f202193b7)

**Comment by arvinxx** [comment]
另外一个产品是哪个？我可以去看看他们的默认模型是哪个。

lobe-chat 使用的默认模型 gpt-3.5-turbo 是 openai 自动更新的版本，当前应该是指向的0613，性能和质量应该是不如1106和0125这两个更新版本的
